### DISCUSSION TITLE
How to highlight current section in top nav menu?

### DISCUSSION THREAD

**Discussion by jaimeiniesta** [saadeghi/daisyui#2828] [Q&A] [answered]
I'm using the daisyUI landing template and the top nav menu goes to sections in the same document.

I want to highlight the current section like in this example

![Captura de pantalla 2024-02-06 a las 13 29 09](https://github.com/saadeghi/daisyui/assets/2629/cc20acd0-9bf5-419e-a0a1-6165b4994c0b)

For me it's enough if it comes selected by the anchor URL or when the section comes into the top of the viewport. I can do that from LiveView or JS but is there a pure daisyUI approach to this?

**Comment by saadeghi** [comment, marked as answer]
> but is there a pure daisyUI approach to this?

I don't think so. Because JS can monitor the scroll position and check which section is currently on screen. But that's not something CSS can do.

**Comment by Avaray** [comment]
That might be possible in crazy way of making six same themes and using `theme-cointroller` 🤪 
https://daisyui.com/components/theme-controller/#theme-controller-using-a-radio-button

That's my 1st idea. But don't do that 😅
